### PR TITLE
feat: implement author-index page

### LINKED ISSUE #194: feat: implement authors index page
https://www.petplace.com/authors/

### PR DESCRIPTION
**Pull request by FentPams**
Fix #194 
Apply updated skeleton to load author card in author-index page.

Test URLs:
- Before: https://main--petplace--hlxsites.hlx.page/authors
- After: https://issue-194--petplace--hlxsites.hlx.page/authors
Screenshot 2023-07-07 at 12 06 48 PM

### CODE REVIEW

File: templates/author-index/author-index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import ffetch from '../../scripts/ffetch.js';
```
Comment (ramboz) [inline templates/author-index/author-index.js:1]: ffetch is probably an overkill here. We will only have a handful of authors so can use a plain fetch and simplify the data retrieval logic

File: templates/author-index/author-index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import { buildBlock } from '../../scripts/lib-franklin.js';
+import { getAuthorImage, meterCalls } from '../../scripts/scripts.js';
+
+async function renderAuthors(authors) {
+  const block = document.querySelector('.cards');
+  block.querySelectorAll('li').forEach((li) => li.remove());
+  for (let i = 0; i < 12; i += 1) {
```
Comment (ramboz) [inline templates/author-index/author-index.js:8]: I'd extract the `12` as a constant at the top to make updating easier

File: templates/author-index/author-index.js [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+function createTemplateBlock(main, blockName) {
+  const section = document.createElement('div');
+
+  const block = buildBlock(blockName, { elems: [] });
+  section.append(block);
+  main.append(section);
+}
+
+export async function loadEager(main) {
+  createTemplateBlock(main, 'pagination');
```
Comment (ramboz) [inline templates/author-index/author-index.js:50]: We probably also don't need pagination if we only ever have ~20 authors

**Comment by FentPams** [comment]
@ramboz Could you take a look of this and give some suggestions.  It can read authors infor from [query-index](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/petplace/authors/[query-index.xlsx](https://adobe.sharepoint.com/:x:/r/sites/HelixProjects/Shared%20Documents/sites/petplace/authors/query-index.xlsx?d=w29529e3cec8344e994777fec7533a9dd&csf=1&web=1&e=R7no6U)?d=w29529e3cec8344e994777fec7533a9dd&csf=1&web=1&e=R7no6U), but I think it should have a better way to improve.  And for the spreedsheet, I added the those manually for test, and trying to  figure out dynamically loading.

**Review by ramboz — APPROVED** [review]

**Pull request merged**